SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py

```diff
@@ -50,6 +52,9 @@
 from Configuration.StandardSequences.Eras import eras
 if eras.fastSim.isChosen():
     hcalSimBlock.hitsProducer = cms.string('famosSimHits')
+if eras.run2_HF_2016.isChosen():
```

Comment:
isChosen is not a good choice
@Dr15Jones please correct me if I'm wrong, and if not please suggest what's the best solution now.